ledger-service/http-json/src/main/resources/logback.xml

```diff
@@ -9,7 +9,7 @@
                 <!-- encoders are assigned the type
                     ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
                 <encoder>
-                    <pattern>%date{"dd-MM-yyyy HH:mm:ss.SSS", UTC} [%thread] %-5level %logger{36} - %msg %replace(, context: %marker){', context: $', ''} %n</pattern>
+                    <pattern>%date{"dd-MM-yyyy HH:mm:ss.SSS", UTC} [%thread] %-5level %logger{36} - %msg%replace(, context: %marker){', context: $', ''} %n</pattern>
```

Comment:
That thing here changes the formatting to instead of this "SomeSuperNiceLoggingText , context: ..." to "SomeSuperNiceLoggingText, context: ...". The space before the `,` annoyed me too much.